libraries/resource_providers/api2600/synergy/hypervisor_manager_provider.rb

```diff
@@ -0,0 +1,20 @@
+# (c) Copyright 2020 Hewlett Packard Enterprise Development LP
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed
+# under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+# CONDITIONS OF ANY KIND, either express or implied. See the License for the
+# specific language governing permissions and limitations under the License.
+
+module OneviewCookbook
+  module API2600
+    module Synergy
+      # HypervisorManager API2600 Synergy provider
+      class HypervisorManagerProvider < OneviewCookbook::API2400::Synergy::HypervisorManagerProvider
```

Comment:
what about force param.? Is that working in chef without changing library file..?

Reply:
Yes, just added the param in examples